Azure/azure-service-operator
# Patch quickstart workflow for devcontainer

What this PR does / why we need it:
This PR patches a few small issues with the devcontainer quickstart workflow.
- Injects environmeent variables from .env file into .devcontainer/Dockerfile and ./Dockerfile so that user does not have to pass secrets in multiple locations. 
- Installs sudo in the devcontainer (needed for downstream mv commands)
- runs "make generate" before build command in the Dockerfile

Special notes for your reviewer:
To Test:
- spin up a new devcontainer by cloning the repo, adding the .env file following README quickstart guidelines. Devcontainer and solution should build with no errors.

If applicable:

 this PR contains documentation
 this PR contains tests
